packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue

```diff
@@ -218,6 +222,7 @@ const floatingPlacement = computed(() =>
     alignEnd: endBoolean.value,
   })
 )
+const sizeStyles = ref<{maxHeight?: string; maxWidth?: string}>({})
```

Comment:
```suggestion
const sizeStyles = ref<StyleValue>({})
```
This should work better

Reply:
That gave me error: Type instantiation is excessively deep and possibly infinite.
CSSProperties worked.